tests/integration/components/add-reminder-test.js

```diff
@@ -0,0 +1,36 @@
+import { moduleForComponent, test, skip } from 'ember-qunit';
+import hbs from 'htmlbars-inline-precompile';
+
+moduleForComponent('add-reminder', 'Integration | Component | add reminder', {
+  integration: true,
+});
+
+test('it renders the add reminder form correctly', function(assert) {
+
+  this.render(hbs`{{add-reminder}}`);
+
+  assert.equal(this.$('label').first().text().trim(), 'Title:');
+  assert.equal(this.$('label:eq(1)').text().trim(), 'Date');
+  assert.equal(this.$('label:eq(2)').text().trim(), 'Notes');
+  assert.equal(this.$('input').length, 2);
+  assert.equal(this.$('textarea').length, 1);
+  assert.equal(this.$('button').length, 1);
+
+});
+
+skip('it should trigger addReminder action on submit', function(assert) {
```

Comment:
This is the error we get when trying to run it

Promise rejected after it should trigger addReminder action on submit: Assertion Failed: You can only unload a record which is not inFlight. `<reminder:null>`@ 113 ms
Source: 	
Error: Assertion Failed: You can only unload a record which is not inFlight. `<reminder:null>`
    at EmberError (http://localhost:7357/assets/vendor.js:29636:21)

I don't know why the record would be in flight because reminder should be coming from the stub function that's almost identical to the one in this example in the docs
<img width="758" alt="screen shot 2017-02-13 at 9 09 00 am" src="https://cloud.githubusercontent.com/assets/21284694/22891298/201b645a-f1cc-11e6-8e94-1f0d773a506f.png">